### PR TITLE
Update root and todo readme files.

### PR DESCRIPTION
**Pull request by verneleem**
Update root and todo-app-react README files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+    One-Click Deployable Apps with<br />
+    <a href="https://dgraph.io/slash-graphql" target="_blank">
+      <img src="https://dgraph.io/assets/images/slashgraphql-logo.svg" alt="Slash GraphQL" />
+    </a><br />
+    A fully managed GraphQL backend service
+  </h1>
+</div>
+
+<p align="center">
+This repository contains sample GraphQL applications powered by Dgraph. <br />As these apps are developed they will be made available for one-click deployment on <a href="https://dgraph.io/slash-graphql" target="_blank">Slash GraphQL</a>.</p>
```
Comment (aaroncarey) [inline README.md:12]:
```suggestion
This repository contains sample GraphQL applications powered by Dgraph database. <br />As these apps are developed they will become available for one-click deployment on <a href="https://dgraph.io/slash-graphql" target="_blank">Slash GraphQL</a>.</p>
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
-# GraphQL Sample Apps
+<div style="padding-top: 5px; padding-bottom: 20px; margin-bottom:20px;">
+  <h1 align="center">
+    One-Click Deployable Apps with<br />
+    <a href="https://dgraph.io/slash-graphql" target="_blank">
+      <img src="https://dgraph.io/assets/images/slashgraphql-logo.svg" alt="Slash GraphQL" />
+    </a><br />
+    A fully managed GraphQL backend service
```
Comment (aaroncarey) [inline README.md:7]:
```suggestion
    A fully-managed GraphQL backend service
```

File: todo-app-react/README.md [hunk 3]
```diff
@@ -1,283 +1,44 @@
...
+    <a href="https://dgraph.io/slash-graphql" target="_blank">
+      <img src="https://dgraph.io/assets/images/slashgraphql-logo.svg" alt="Slash GraphQL" />
+    </a><br />
+    A fully managed GraphQL backend service
+  </h2>
+</div>
+
+<h3 align="center"><a href="https://slash.dgraph.io/_/one-click?app=todo" target="_blank">Deploy Now</a> for free!</h3>
+
+To-Do is a sample app that lets uers manage the tasks on their personal to-do list. Developers can see how to use react hooks with apollo client to easily create, read, update, and delete To-Do items. Deploying this app on Slash deploys both the back-end database service and a front-end react app in a single click, no credit card required. To learn more about this sample app, see: [Building a To-Do List React App with Dgraph](https://dgraph.io/blog/post/building-todo-list-react-dgraph/)
```
Comment (aaroncarey) [inline todo-app-react/README.md:14]:
```suggestion
To-Do is a sample app that lets users manage the tasks on their personal to-do list. This app demonstrates how to use React hooks with an Apollo client to easily create, read, update, and delete to-do list items. Deploying this app on Slash GraphQL deploys both the back-end database service and a front-end React app in a single click, no credit card required. To learn more about this sample app, see: [Building a To-Do List React App with Dgraph](https://dgraph.io/blog/post/building-todo-list-react-dgraph/)
```

File: todo-app-react/README.md [hunk 4]
```diff
@@ -1,283 +1,44 @@
...
-
-## Screenshots
+<div style="padding-top: 5px; padding-bottom: 10px;">
+  <h1 align="center">ToDo React App</h1>
+  <h2 align="center">
+    A One-Click Deployable App with<br />
+    <a href="https://dgraph.io/slash-graphql" target="_blank">
+      <img src="https://dgraph.io/assets/images/slashgraphql-logo.svg" alt="Slash GraphQL" />
+    </a><br />
+    A fully managed GraphQL backend service
```
Comment (aaroncarey) [inline todo-app-react/README.md:8]:
```suggestion
    A fully-managed GraphQL backend service
```

**Review by aaroncarey — APPROVED** [review]
Approved with a few small changes.

**Review by tstreamDOTh — APPROVED** [review]
lgtm

**Comment by aaroncarey** [comment]
Still looks good from my perspective!

**Comment by verneleem** [comment]
Well, none of us anticipated Github's darkmode. Has anyone found a way yet to change a picture depending on color mode in md? Here is how this looks in darkmode:

image
 
Verses regular light mode:

image

**Comment by verneleem** [comment]
Well, there is a discussion in progress about how to support multiple images sources depending on mode. https://github.community/t/support-theme-context-for-images-in-light-vs-dark-mode/147981?u=verneleem

**Pull request merged**